go.mod

```diff
@@ -7,6 +7,11 @@ require (
 	github.com/mongodb-forks/digest v1.0.5
 )
 
+require (
+	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
+	github.com/hashicorp/go-retryablehttp v0.7.5 // indirect
+)
```

Comment:
can this be avoided? hashicorp libs are painful because they are MPL and get flagged in snyk and then it requires legal to approve

Reply:
yes. I will create dedicated go.mod file for examples in this PR.

Reply:
Done